packages/core/src/CML/coreToCml/coreToCml.ts

```diff
@@ -90,11 +90,12 @@ export const tokenMap = (scope: ManagedFreeableScope, map: Cardano.TokenMap) =>
 
 export const value = (scope: ManagedFreeableScope, { coins, assets }: Cardano.Value): Value => {
   const result = scope.manage(Value.new(scope.manage(BigNum.from_str(coins.toString()))));
+
   if (!assets) {
     return result;
   }
   if (assets.size > 0) {
-    result.set_multiasset(tokenMap(scope, assets));
+    result.set_multiasset(scope.manage(tokenMap(scope, assets)));
```

Comment:
I'm a bit confused about why this changes the behavior when the returned object is already being managed here: https://github.com/input-output-hk/cardano-js-sdk/blob/a29639715f47d7d5cab7d7aa667581bb1030620f/packages/core/src/CML/coreToCml/coreToCml.ts#L63